RxJavaTemplate/buildSrc/src/main/java/Dependencies.kt

```diff
@@ -0,0 +1,136 @@
+object AppModule {
+    val implementations = listOf(
+        // Kotlin stdlib
+        KOTLIN_STDLIB_JDK7,
+        KOTLIN_STDLIB,
+        // Android components
+        ANDROIDX_CORE,
+        ANDROIDX_FRAGMENT_KTX,
+        ANDROIDX_LEGACY_SUPPORT,
+        ANDROIDX_APPCOMPAT,
+        ANDROIDX_RECYCLERVIEW,
+        ANDROID_MATERIAL,
+        ANDROIDX_CONSTRAINT_LAYOUT,
+        // Android navigation
+        ANDROIDX_NAVIGATION_RUNTIME_KTX,
+        ANDROIDX_NAVIGATION_FRAGMENT_KTX,
+        ANDROIDX_NAVIGATION_UI_KTX,
+        // Rx
+        RX_BINDING,
+        RX_BINDING_CORE,
+        // Lifecycle
+        ANDROIDX_LIFECYCLE_VIEWMODEL_KTX,
+        ANDROIDX_LIFECYCLE_EXTENSIONS,
+        ANDROIDX_LIFECYCLE_COMMON_JAVA8,
+        ANDROIDX_LIFECYCLE_RUNTIME,
+        // Hilt
+        HILT,
+        // Others
+        ANDROIDX_MULTIDEX,
+        TIMBER,
+        GLIDE,
+        RX_PERMISSION
+    )
+
+    val debugImplementations = listOf(
+        ANDROIDX_FRAGMENT_TESTING,
+        CHUCKER
+    )
+
+    val releaseImplementations = listOf(
+        CHUCKER_NO_OP
+    )
+
+    val kapts = listOf(
+        ANDROIDX_LIFECYCLE_COMPILER,
+        GLIDE_COMPILER,
+        HILT_COMPILER
+    )
+
+    val androidTestKapts = listOf(
+        HILT_ANDROID_COMPILER
+    )
+
+    val androidTestImplementations = listOf(
+        // Test runner
+        ANDROIDX_TEST_RUNNER,
+        ANDROIDX_TEST_RULES,
+        // Espresso
+        ANDROIDX_TEST_ESPRESSO_CORE,
+        ANDROIDX_TEST_ESPRESSO_INTENTS,
+        ANDROIDX_TEST_ESPRESSO_CONTRIB,
+        ANDROIDX_TEST_ESPRESSO_WEB,
+        // Hilt
+        HILT_ANDROID_TESTING,
+        // Mockito
+        MOCKITO_INLINE
+    )
+
+    val testImplementations = listOf(
+        JUNIT,
+        // kluent
+        KLUENT_ANDROID,
+        // Mockito
+        MOCKITO_KOTLIN,
+        // Robolectric
+        ROBOLECTRIC
```

Comment:
I wonder do we really need these comments? I feel like the constant name is explicit itself better. 🤔  (The same idea as at other variables as well.)

Reply:
@lydiasama I was supposed to add the comment to group the dependencies with the same package. 🤔  I will remove the comment if there is only one dependency from the package. What do you think?

Reply:
@lydiasama I updated to remove all the unnecessary comments 😄